community/modules/scheduler/schedmd-slurm-gcp-v6-controller/modules/slurm_files/scripts/tests/test_resume.py

```diff
@@ -74,11 +74,11 @@ def test_group_nodes_bulk(mock_create_placements, mock_tpu):
         "t": TstNodeset(nodeset_name="t"),
       },
       partitions={
-        "p1": TstPartition(
+        "p1": dict(
```

Comment:
Why not use the Partition class?

Reply:
Because it mock raw config that uses Dict, the `Partition` class is not integrated into config yet. It's a follow up action.